homeassistant/components/fibaro.py

```diff
@@ -395,5 +395,7 @@ def device_state_attributes(self):
         except (ValueError, KeyError):
             pass
 
+        attr['fibaro_room'] = self.fibaro_device.room_name
+        attr['fibaro_hub'] = self.controller.hub_serial
```

Comment:
We shouldn't do this. Just implement device info and it can be tracked via the device registry using `via_hub`.